kontrakter/kriminalomsorg/returPersonundersoekelse/1.0/returPersonundersoekelse.schema.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://kriminalomsorg.no/kriminalomsorg/1.0/returPersonundersoekelse",
+  "description": "Retur som svarer ut en rekvirert personundersøkelse. Inneholder normalt den ferdige PeU-rapporten (PDF). Sendes fra kriminalomsorgen til politiet. Schema versjon 2020-12",
+  "type": "object",
+  "properties": {
+    "forsendelse": {
+      "$ref": "#/definitions/forsendelse",
+      "description": "avsender og mottaker"
+    },
+    "hovedStraffesaksnummer": {
+      "$ref": "#/definitions/straffesaksnummer",
+      "description": "BL Saksbehandlersak er hovedsaken i et sakskompleks (som er 1 eller flere straffesaker)"
+    },
+    "rekvisisjonPersonundersoekelseId": {
+      "$ref": "#/definitions/unikId",
+      "description": "ID  på den tidligere rekvisisjonen kriminalomsorgen nå svarer ut. "
+    },
+    "returPersonundersoekelseId": {
```

Comment:
Kunne i grunn bare hetet `personundersoekelseId`? 🤷 

..i så fall også naturlig å endre navnet på mappen og andre steder det er brukt

Reply:
Lar den stå slik den var i denne omgang. 

Har ikke veldig sterke følelser for dette, men heller til at svar/retur i en PeU sak av og til er på form "returnert da det ikke var mulig å utføre undersøkelsen (siktede nekter møte, nekter undersøker info m.m.)". Og da er det kanskje litt misvisende å kalle det svaret for en personundersøkelse. Men igjen, ikke veldig viktig - kan endres om andre føler sterkere for dette :)